xscen/indicators.py

```diff
@@ -75,16 +76,18 @@ def compute_indicators(
       Can also be only the "stem", if translations and custom indices are implemented.
       Can be the indicator module directly, or a sequence of indicators or a sequence of
       tuples (indicator name, indicator) as returned by `iter_indicators()`.
+    periods : list
+      list of [start, end] of the contiguous periods to be evaluated, in the case of disjointed datasets.
```

Comment:
Not super clear what you mean here...

I think contiguous periods would mean periods that have the same border (eg. 2005-2010, 2010-2015). I think we want contiNuous periods ( no holes inside the periods).
```suggestion
      list of [start, end] of continuous periods over which to compute the indicators. This is needed when the time axis of ds contains some jumps in time.
```